node/README.md

```diff
@@ -189,11 +189,14 @@ things to note:
 
 Both of these cases are handled by the electron-webpack-quick-start example.
 
-### Non-patented ffmpeg
+### ffmpeg support
+
+If your app needs separate ffmpeg archive, add `--electron-ffmpeg=archive` option
```

Comment:
Might be worth noting where the extra archive gets placed and why you'd need it (e.g. particular packages / use cases and the errors that might arise otherwise).

Reply:
I'm not sure what and when exactly require it. Looks like it's an opt-in in electron-downloader, probably for apps that want to create patent-free distribution themselves. I've stumbled upon it while building Theia (VS Code fork).